--- lalxml-1.2.4/src/LALXML.c.orig	2015-03-27 05:14:39.000000000 +0100
+++ lalxml-1.2.4/src/LALXML.c	2024-06-09 16:58:09.395143737 +0200
@@ -29,6 +29,7 @@
 #include <libxml/tree.h>
 #include <libxml/xpath.h>
 #include <libxml/xpathInternals.h>
+#include <libxml/xmlsave.h>
 #include <libxml/xmlschemas.h>
 
 #include <lal/LALStdio.h>
--- lalxml-1.2.4/gnuscripts/lalsuite_gccflags.m4~	2016-09-13 16:51:01.000000000 +0200
+++ lalxml-1.2.4/gnuscripts/lalsuite_gccflags.m4	2025-05-11 15:58:41.169998418 +0200
@@ -30,7 +30,6 @@
       -g3
       -Wall
       -W
-      -Werror
       -Wmissing-prototypes
       -Wstrict-prototypes
       -Wshadow
